tests/vibrato_tests.ts

```diff
@@ -67,11 +69,11 @@ function harsh(options: TestOptions, contextBuilder: ContextBuilder): void {
         { str: 4, fret: 9 },
       ],
       duration: 'h',
-    }).addModifier(new Vibrato().setVibratoCode(0xeae2), 0),
+    }).addModifier(new Vibrato().setVibratoCode(SMuFLCodePoint.wiggleVibratoLargeFastest /* 0xeae2 */), 0),
```

Comment:
@rvilarl Should we be relying on code points? Why not just accept the unicode character directly?

In this case, we would use: `'\ueae2'`